### DISCUSSION TITLE
Downloads are broken

### DISCUSSION THREAD

**Discussion by Eismann** [thirtythreeforty/neolink#306] [General]
Hi,

i want try the installation for my Reolink E1 on a Raspberry but the link

Debian aarch64 (Raspberry Pi 64-bit) ([download](https://nightly.link/thirtythreeforty/neolink/workflows/build/master/release-arm64-buster.zip))

 is broken.

"Error 404 – Not Found
GitHub produced an error for the download of artifact #267903006.

Usually this means that the artifact has expired (>90 days).

Check on GitHub: https://github.com/thirtythreeforty/neolink/suites/6902331785/artifacts/267903006"

Can you pls fix this error?

Thanks :-)

**Comment by thirtythreeforty** [comment]
Sorry about that.  Indeed it has expired.

Pushed 58e74fc which should let us regenerate them one-click in the future. Of course this push will regenerate it this time, but in case it's needed for the future we'll have it :smile:

**Reply by QuantumEntangledAndy** [reply]
Is there a reason we don't use releases and instead use assets? (which require a GitHub account I think). I have an old PR that builds the releases automatically with a commit it tagged if that would help I could update it.

**Reply by thirtythreeforty** [reply]
We can definitely start issuing releases. Development was moving so fast initially that I could never point to a "stable" version - I always wanted you to use the latest.

Yeah it does require a GitHub account (which is dumb) but I have the readme hooked up to nightly.link to alleviate that requirement.
